msteams-platform/concepts/device-capabilities/people-picker-capability.md

```diff
@@ -1,57 +1,66 @@
-* This control allows you to search for and select users within a chat, channel, or the entire organization.
-* People Picker helps with scenarios involving task assignment, tagging, notifying a user.
-* You can use this readily available control in your web app. It saves the effort and time significantly to build such a control on your own.
+## Advantages of integrating People Picker
 
-You must call the `selectPeople` API to integrate People Picker control in your Teams app. For effective integration, you must have an understanding of [code snippet](#code-snippet) for calling the API.
-It is important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your web app.
+* People Picker control works on all Teams platforms, such as task module, a chat, channel, meeting tab, and personal app.
+* The control allows the user to search and select user in a chat, channel, or the entire organization.
+* People Picker helps with scenarios involving task assignment, tagging and notifying a user.
+* The control can be used in both web and desktop, as it saves the effort and time significantly to build a similar control.
 
-> [!NOTE]
-> Currently, Microsoft Teams support for People Picker is available for mobile clients only.
+To integrate People Picker control in your Teams app select the API, `selectPeople`. To integrate and call the API, user must know about [code snippet](#code-snippet). User also needs familiarity with [API response errors](#error-handling) to handle any errors in web and desktop app.
 
 ## `selectPeople` API
 
-`selectPeople` API enables you to add Teams native `People Picker input control` to your web apps.  
-The API description is as follows:
+The `selectPeople` API enables users to add Teams native `People Picker input control` to web and desktop apps. The following table provides description of the API:
 
-| API      | Description  |
+| API | Description  |
 | --- | --- |
-|**selectPeople**|Launches a People Picker and allows the user to search and select one or more people from the list.<br/><br/>This API returns the ID, name and email address of selected users to the calling web app.<br/><br/>In case of a personal app, the control searches across the organization. If the app is added to a chat or channel, then the search context is configured depending on the scenario. The search is restricted within the members of that chat, channel, or made available across the organization.|
+|`selectPeople`| • Launches People Picker and allows the user to search and select one or more people from the list.<br/>  • This API returns the ID, name and email address of selected users to the web or desktop app.<br/> • In case of a personal app, the control searches across the organization. If the app is added to a chat or channel, then the search context is configured depending on the scenario. The search is restricted within the members of that chat, channel, or made available across the organization.|
```

Comment:
```suggestion
|`selectPeople`| • Launches People Picker and allows the user to search and select one or more people from the list.<br/>  • This API returns the ID, name and email address of selected users to the web app.<br/> • In case of a personal app, the control searches across the organization. If the app is added to a chat or channel, then the search context is configured depending on the scenario. The search is restricted within the members of that chat, channel, or made available across the organization.|
```